server.go

```diff
@@ -235,6 +236,8 @@ func (s *Server) handleRequest(w http.ResponseWriter, r *http.Request, route *Ro
 	resBuilder.SetLayout(results[0])
 	resBuilder.SetHeaders(results[0].HeadersWithoutProxyHeaders(), results)
 	resBuilder.SetFragments(results[1:])
+	elapsed := time.Since(start_time)
```

Comment:
```suggestion
	elapsed := time.Since(startTime)
```

Minor, but Go conventionally uses camel case for variable names